### PR TITLE
Update package versions, add test scripts, add package specific preview setup

### PR DESCRIPTION
**Pull request by peterhass**
https://favro.com/organization/aee06c3fe6503c804f507dd9/22c630cc02b09f2d6f98b8b3?card=Goo-32133

### CODE REVIEW

File: package-templates/react/.storybook/.gitkeep [hunk 1]
```diff
@@ -0,0 +1 @@
+keep
```
Comment (suprMax) [inline package-templates/react/.storybook/.gitkeep:1]: empty file works as well

File: scripts/run-tests [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# expects cwd to be subpackage root
+
+NODE_ENV=test TZ='GMT' mocha \
+  --exclude "./node_modules/**" \
```
Comment (suprMax) [inline scripts/run-tests:5]: would just `**/node_modules` work?

**Review by ozerovs — APPROVED** [review]
LGTM

File: scripts/run-tests (hunk 2 above)
Comment (peterhass) [reply scripts/run-tests:5]: tried it and nope - doesn't work

no idea why

**Review by suprMax — APPROVED** [review]

**Pull request merged**